user/models.py

```diff
@@ -57,9 +59,12 @@ def create(cls, access_token=None, **kwargs):
         username = kwargs.get("sub")
         user = User.objects.create(username=username, **kwargs)
         client = ApiClient(access_token)
-        cloud = client.collaboration_api.create_cloud(
-            data={"name": f"{user.first_name} {user.last_name}"}
-        )
+        try:
```

Comment:
This call should always work. If it does not, it's a real 500 and we should be alerted